src/components/Common/Badge/index.tsx

```diff
@@ -52,17 +52,23 @@ const Badge: React.FC<BadgeProps> = ({
     badgeStyle.push(className);
   }
 
-  if (url) {
+  if (href?.includes('://')) {
     return (
       <a
-        href={url}
+        href={href}
         target="_blank"
         rel="noopener noreferrer"
         className={badgeStyle.join(' ')}
       >
         {children}
       </a>
     );
+  } else if (href) {
+    return (
+      <a href={href} className={badgeStyle.join(' ')}>
+        {children}
+      </a>
```

Comment:
Since these can be in-app links now, we should use `<Link>` around this to allow client-side routing.